rtc/configuration.md

```diff
@@ -8,361 +8,359 @@ keywords: rtc configuration
 
-### Example of providing custom user details for `rtc_user_connected`
+#### User Details
 
-```js
-tinymce.init({
-  selector: 'textarea',  // change this value according to your HTML
-  plugins: 'rtc',
-  rtc_user_connected: ({userId, caretNumber, custom}) => {
-    console.log('Connected', userId, caretNumber, custom);
-  }
-})
-```
+This is a copy of the object returned by [`rtc_user_details_provider`](#rtc_user_details_provider). RTC only uses the `fullName` field, but the entire object will be cloned and passed to `rtc_client_connected`.
+
+#### Client ID
+
+This is a unique identifier, generated by the RTC protocol, that can be used to differentiate between the same user connecting multiple times.
+
+#### Caret Number
+
+This number corresponds to one of [the 8 colours defined in TinyMCE CSS](https://github.com/tinymce/tinymce/blob/master/modules/oxide/src/less/theme/content/rtc/rtc.less#L1-L8). TinyMCE supports 8 distinct caret colors. If more than 8 clients connect to a session, the numbers will be reused.
+
+
+A custom skin is required to change these colours, and no more than 8 are supported in this release. For more information on creating a custom skin, see the [Customizing the Editor UI]({{site.baseurl}}/general-configuration-guide/customize-ui/) article.
 
-## `rtc_user_disconnected`
+#### Client information
 
-This option allows applications to track when a user leaves the session. This is useful for keeping a list or array of connected users up-to-date.
+This is a copy of the [`rtc_client_info`](#rtc_client_info) data from the remote user's editor configuration.
+
+> **Caution**: client information data has no authenticity guarantee as it comes from a remote object. {{site.companyname}} recommends only using the client information data for status flags, to obtain authentic information leverage the [`rtc_user_details_provider`](#rtc_user_details_provider) API via the user details field.
 
 **Type:** `Function`
 
 **Required:** no
 
-### Input fields for `rtc_user_disconnected`
+#### Input fields for `rtc_client_connected`
 
 | Field | Type | Description |
 |-------|:----:|-------------|
-| `userId` | `string` | This is the unique user ID of the disconnecting user. |
-| `caretNumber` | `integer` | The user's caret number 1-8 (helpful to disambiguate when a user connects multiple times). |
-| `custom` | `object` | Custom data passed out from the other clients `rtc_custom_user_details` function. If none are provided, this will be an empty object. |
+| `userId` | `string` | The user's unique ID |
+| `userDetails` | `object` | User details object. If none is configured, this will be an empty object. |
+| `clientId` | `string` | The unique ID for this client connection |
```

Comment:
`this` seems more correct, because there will be a new one for every single remote client that joins the collaboration session? there is no "current" client connection.

Reply:
`this` seems too general. I was trying to find something more specific.

Reply:
I'll go with `the new client connection`, see how that looks